google/ggrc-core
# GGRC-8166 Implement notification switcher setting

# Issue description

It should be possible to disable notifications sending via settings during application deployment.

# Steps to test the changes

1. Deploy application with `NOTIFICATIONS_ENABLED=""` and `GCALENDAR_ENABLED=""` settings;
2. Try import-export;
3. Try mentioning a person;
4. Try marking a program as _reviewed_ / _unreviewed_;
5. Try leaving comments;
6. Try creating cycle tasks and assign it to people.

_Expected result: no emails should be sent and no events should appear in the user's google calendar._

# Solution description

*Briefly explain the solution or add a link to the technical design document (this will help the reviewer confirm the code does what you intend). Feel free to also mention alternative approaches that didn't work out.*

# Sanity checklist




# PR Review checklist
